src/python/geoclaw/surge/storm.py

```diff
@@ -25,21 +25,23 @@
     - GeoClaw (fully)
     - ATCF (reading only)
     - HURDAT (reading only)
+    - IBTrACS (reading only)
     - JMA (reading only)
     - IMD (planned)
     - tcvitals (planned)
 """
 
 from __future__ import print_function
 from __future__ import absolute_import
-
 from six.moves import range
 
 import warnings
 import sys
 import os
 import argparse
 import datetime
+import xarray as xr
```

Comment:
Can you put both the `xarrays` and `pandas` modules into a try block (maybe in the `read_ibtracs`) so that if someone is not reading the IBTrACS data they do not have to install `xarray` and `pandas`.